gulpfile.js

```diff
@@ -41,30 +41,18 @@ var gulp = require('gulp'),
     };
 
 /* CSS - LESS */
-function processCss(inputStream, filename, taskType) {
-    // Grab vendor CSS
-    var vendorStream = gulp.src(paths.styles.vendor.src + '*.css')
-        .pipe($.plumber(function(error) {
-            $.util.log($.util.colors.red('Error (' + error.plugin + '): ' + error.message));
-            this.emit('end');
-        }));
-
-    // Compile LESS stream
-    inputStream = inputStream
+function processCss(inputStream, taskType) {
```

Comment:
No more need to process vendor css stream separately, simply return the processed the inputStream